### PR TITLE
doc: Fix documentation issues

### PR DESCRIPTION
**Pull request by adeatcu-ionos**
## Checklist

- [ ] PR name added as appropriate (e.g. `feat:`/`fix:`/`doc:`/`test:`/`refactor:`)
- [ ] Tests added or updated
- [ ] Documentation updated
- [ ] Changelog updated and version incremented (label: upcoming release)
- [ ] Github Issue linked if any
- [ ] Jira task updated

### CODE REVIEW

File: docs/resources/server.md [hunk 1]
```diff
@@ -22,6 +22,7 @@ data "ionoscloud_image" "example" {
     type                  = "HDD"
     cloud_init            = "V1"
     image_alias           = "ubuntu:latest"
+    location              = "us/ewr"
```
Comment (cristiGuranIonos) [inline docs/resources/server.md:25]: shouldn't this match the datacenter location? us/las?

File: docs/resources/volume.md [hunk 2]
```diff
@@ -19,7 +19,8 @@ A primary volume will be created with the server. If there is a need for additio
 data "ionoscloud_image" "example" {
     type                  = "HDD"
     cloud_init            = "V1"
-    location              = "us/las"
+    image_alias           = "ubuntu:latest"
+    location              = "us/ewr"
```
Comment (cristiGuranIonos) [inline docs/resources/volume.md:23]: same as above

File: docs/resources/server.md (hunk 1 above)
Comment (adeatcu-ionos) [reply docs/resources/server.md:25]: @cristiGuranIonos no, it doesn't matter where the image is coming from.

File: docs/resources/volume.md (hunk 2 above)
Comment (adeatcu-ionos) [reply docs/resources/volume.md:23]: @cristiGuranIonos same as above, it doesn't matter when speaking about the image.

File: docs/resources/server.md (hunk 1 above)
Comment (cristiGuranIonos) [reply docs/resources/server.md:25]: what if it's not available in the location?

Comment (cristiGuranIonos) [reply docs/resources/server.md:25]: I mean, if it's an example we should keep the location the same in the entire example, even if ubuntu:latest is available in all locations.

Comment (adeatcu-ionos) [reply docs/resources/server.md:25]: @cristiGuranIonos `what if it's not available in the location?` then the image will not be fetched, I don't think that we can reach a configuration to ensure that a image will always be available, whether we talk about `location`, `image_alias` or `type`. 

`if it's an example we should keep the location the same in the entire example, even if ubuntu:latest is available in all locations.` ok, I will change the location

Comment (adeatcu-ionos) [reply docs/resources/server.md]: @cristiGuranIonos done

File: docs/resources/volume.md (hunk 2 above)
Comment (adeatcu-ionos) [reply docs/resources/volume.md]: @cristiGuranIonos done

**Review by cristiGuranIonos — APPROVED** [review]

**Pull request merged**